packages/discovery-provider/integration_tests/tasks/user_bank_mock_transactions.py

```diff
@@ -1413,6 +1415,451 @@
+                        1009200,
+                        953185920,
+                        1,
+                        1141440,
+                    ],
+                    "innerInstructions": [
+                        {
+                            "index": 0,
+                            "instructions": [
+                                {
+                                    "programIdIndex": 6,
+                                    "accounts": [0, 1, 3],
+                                    "data": "R7r7mFYpVnfogxGmgKgoouv7CPupJtRDwQVv3vdrK5Jc6WMMKQSCJRVvFjiJTnjDsSdhQknmHiuv3mTonKg92uBjwC9jzpaHzmZ3MtdyZEQ5ThmZ7DHSxrqKFhkdj5A7VGKeVhxp9ooWpZQMUQaD7oVU1EgZpvj5SmE",
+                                    "stackHeight": None,
+                                },
+                                {
+                                    "programIdIndex": 5,
+                                    "accounts": [1, 2, 3, 4],
+                                    "data": "2",
+                                    "stackHeight": None,
+                                },
+                            ],
+                        }
+                    ],
+                    "logMessages": [
+                        "Program {CLAIMABLE_TOKENS_PDA} invoke [1]",
+                        "Program log: Instruction: CreateTokenAccount",
+                        "Program 11111111111111111111111111111111 invoke [2]",
+                        "Program 11111111111111111111111111111111 success",
+                        "Program TokenkegQfeZyiNwAJbNbGKPFXCWuBvf9Ss623VQ5DA invoke [2]",
+                        "Program log: Instruction: InitializeAccount",
+                        "Program TokenkegQfeZyiNwAJbNbGKPFXCWuBvf9Ss623VQ5DA consumed 3272 of 178721 compute units",
+                        "Program TokenkegQfeZyiNwAJbNbGKPFXCWuBvf9Ss623VQ5DA success",
+                        "Program {CLAIMABLE_TOKENS_PDA} consumed 25212 of 200000 compute units",
+                        "Program {CLAIMABLE_TOKENS_PDA} success",
+                    ],
+                    "preTokenBalances": [],
+                    "postTokenBalances": [
+                        {
+                            "accountIndex": 1,
+                            "mint": WAUDIO_MINT,
+                            "uiTokenAmount": {
+                                "uiAmount": None,
+                                "decimals": 8,
+                                "amount": "0",
+                                "uiAmountString": "0",
+                            },
+                            "owner": WAUDIO_PDA,
+                        }
+                    ],
+                    "rewards": [],
+                    "loadedAddresses": {"writable": [], "readonly": []},
+                },
+                "version": "legacy",
+                "blockTime": 1644007841,
+            },
+            "id": 0,
+        }
+    )
+)
+
+mock_valid_waudio_transfer_between_user_banks = GetTransactionResp.from_json(
```

Comment:
Might be helpful to add a comment at the top of some of these transactions to say what exactly is happening. I.e. this one seems to be transferring 100 audio?

Reply:
👍